app/models/container_node.rb

```diff
@@ -115,4 +114,12 @@ def external_logging_path
     index = ".operations.*"
     EXTERNAL_LOGGING_PATH % {:index => index, :query => query}
   end
+
+  def disconnect_inv
+    return if ems_id.nil?
```

Comment:
I think you are right. Actually, we might need to change that condition in all other archived containers entities?